app/api/server/v1/channels.js

```diff
@@ -577,6 +578,11 @@ API.v1.addRoute('channels.members', { authRequired: true }, {
 
 		const { offset, count } = this.getPaginationItems();
 		const { sort = {} } = this.parseJsonQuery();
+		const { status, username, name } = this.queryParams;
+
+		check(status, Match.Maybe([String]));
+		check(username, Match.Maybe(String));
+		check(name, Match.Maybe(String));
 
 		const subscriptions = Subscriptions.findByRoomId(findResult._id, {
 			fields: { 'u._id': 1 },
```

Comment:
Probably not related to your PR, but, why is the offset/count passed to the subscriptions and not to the user's list? (which is the returned value) 👀

Reply:
right @KevLehman .. I have changed it now though =)